asv_bench/benchmarks/io/csv.py

```diff
@@ -76,6 +71,27 @@ def time_skiprows(self, test_filenames, shape, skiprows):
         )
 
 
+class TimeReadCsvTrueFalseValues(BaseReadCsv):
```

Comment:
The test for OmniSci and Ray is very similar, try to find a way to avoid code duplication.

Reply:
Ok, single implementation was used by specifying `execute` method for different backends.